config/environment.rb

```diff
@@ -3,3 +3,5 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+require 'carrierwave/orm/activerecord'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.